tests/functional/test_install.py

```diff
@@ -486,6 +487,41 @@ def test_install_package_with_root(script, data):
     assert root_path in result.files_created, str(result)
 
 
+def test_install_package_with_prefix(script, data):
+    """
+    Test installing a package using pip install --prefix
+    """
+    prefix_path = script.scratch_path / 'prefix'
+    result = script.pip(
+        'install', '--prefix', prefix_path, '-f', data.find_links,
+        '--no-index', 'simple==1.0',
```

Comment:
Maybe you should add the `--no-binary simple` option to make sure/be clear that this tests a sdist install and not a wheel install (through wheel cache).